src/lib.rs

```diff
@@ -105,6 +105,7 @@ pub enum Action {
     BuyerTookOrder,
     RateUser,
     CantDo,
+    MessageReceived,
```

Comment:
could you change this from `MessageReceived` ? it's always a message, `Received`  is more concise

Reply:
Yes for sure, it was just my idea to avoid Ok maybe that was too similar to Rust Ok and can create confusion.

Let's go with Received!